DatadogAlamofireExtension.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = "DatadogAlamofireExtension"
-  s.version      = "2.10.0"
+  s.version      = "2.11.0"
```

Comment:
🤔 Shouldn't it be
```diff
-  s.version      = "2.10.1"
+  s.version      = "2.11.0"
```
here? Aren't we missing the hotfix change?

Reply:
nope, we backported the change from develop

original PR: https://github.com/DataDog/dd-sdk-ios/pull/1807
backport: https://github.com/DataDog/dd-sdk-ios/pull/1810